nemo_curator/utils/script_utils.py

```diff
@@ -392,67 +408,30 @@ def parse_distributed_classifier_args(
             description,
             formatter_class=argparse.ArgumentDefaultsHelpFormatter,
         )
-        parser = ArgumentHelper(parser).add_distributed_args()
+        argumentHelper = ArgumentHelper(parser)
+        argumentHelper.add_distributed_args()
+        argumentHelper.add_arg_input_data_dir(required=True)
+        argumentHelper.add_arg_output_data_dir(help="The path of the output files")
+        argumentHelper.add_arg_input_file_type()
+        argumentHelper.add_arg_input_file_extension()
+        argumentHelper.add_arg_output_file_type()
+        argumentHelper.add_arg_input_text_field()
+        argumentHelper.add_arg_enable_spilling()
+        argumentHelper.add_arg_set_torch_to_use_rmm()
+        argumentHelper.add_arg_batch_size(
```

Comment:
Decided to clean this up too because i was in the same fine, @ryantwolf , please take a look.